plugin/trino-hive/src/test/java/io/trino/plugin/hive/parquet/ParquetTester.java

```diff
@@ -768,6 +781,42 @@ private static void writeParquetColumnTrino(
         pageBuilder.declarePositions(size);
         writer.write(pageBuilder.build());
         writer.close();
+        writer.validate(writeValidation -> {
```

Comment:
Are there any other tests (e.g. in ORC) that we could add? Would validator work with nested types, complex structures?

Reply:
As far as I could tell ORC tests validation by using it `OrcTester` and in TestHiveFileFormats. I've done the same by always having validation in `ParquetTester` and in `TestHiveFileFormats#testOptimizedParquetWriter`. If you see more scope for testing it, let see know.